src/executeAutomagically.ts

```diff
@@ -103,6 +104,7 @@ export const executeAutomagically = async ({
         testTargetId,
         environmentName,
         variablesToOverwrite: variablesToOverwriteObject,
+        tags,
```

Comment:
uno unit testo?

Reply:
i will buy you a beer if this unit test provides any value ;)